app/models/ahoy/visit.rb

```diff
@@ -2,7 +2,7 @@ module Ahoy
   class Visit < ApplicationRecord
     self.table_name = "ahoy_visits"
 
-    has_many :events, class_name: "Ahoy::Event"
+    has_many :events, class_name: "Ahoy::Event", dependent: :destroy
```

Comment:
Hm, I feel like this is something the ahoy gem _should_ just do for us? It's weird to me that it [doesn't](https://github.com/ankane/ahoy/blob/master/test/internal/app/models/ahoy/visit.rb#L4)?

Reply:
It might be because they want the implementor to decide what to do? But yeah, it's weird to me they don't automatically add the foreign key between the two tables at least